geopandas/tests/test_plotting.py

```diff
@@ -1528,6 +1528,12 @@ def test_pandas_kind(self, kind, fig_test, fig_ref):
                 kwargs = {"y": "y"}
             elif kind in _xy_kinds:
                 kwargs = {"x": "x", "y": "y"}
+                # For the hexbin plot, increase the gridsize to a value
+                # reasonable relatively to the spacing between the data points,
+                # in order to draw fewer hex cells and reduce the duration of
+                # the test, especiallly the comparison in pdf format.
+                if kind == "hexbin":
```

Comment:
```suggestion
                if kind == "hexbin":  # increase gridsize to reduce duration
```

No need to be so verbose :).

Reply:
Yup indeed!